.changeset/slimy-cups-protect.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Does `shared` need to be here too since we updated some code in index.ts?

Reply:
@andrewortwein good question! That's not needed because the shared package is internal and is not published to npm. Any changes to it will be accounted for in the release of either the pw or cy package. We should see if there's a good way to exclude that from the list when generating a changeset.

Reply:
Or better, if the shared package is changed, it should automatically cause a release of both pw and cy.

Reply:
Oooh, right, duh. Nothing to see here. Carry on.

Reply:
Good point @andrewortwein! Is there anything you'd recommend adding to our [dev doc](https://github.com/chromaui/chromatic-e2e/blob/main/DEVELOPMENT.md) to make it more clear what packages are versioned/how the packages depend on each other?

Reply:
I think most of the confusion can just lie in the fact that `shared` shows up as an option when running `yarn changeset`. I suppose, near the top of the doc where we call out the three packages, that we could emphasize that "shared is not actually published anywhere".

Reply:
#167